README.md

```diff
@@ -48,7 +52,10 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+
+town[:residents].delete "Belle"
+town[:castle][:guests] << "Belle"
+
```

Comment:
:+1: maybe consider creating ```belle``` as a variable while deleting her and just adding that?